addons/mail/static/tests/helpers/mock_server/models/discuss_channel_member.js

```diff
@@ -78,4 +78,27 @@ patch(MockServer.prototype, {
         const [member] = this.getRecords("discuss.channel.member", [["id", "in", ids]]);
         return this._mockResPartnerMailPartnerFormat([member.partner_id]).get(member.partner_id);
     },
+    /**
+     * Simulates the '_channel_fold' method on `discuss.channel.member`.
+     *
+     * @private
+     * @param {number} id
+     * @param {state} [state]
+     * @param {number} [state_count]
+     */
+    _mockDiscussChannelMember__channelFold(id, state, state_count) {
+        const [member] = this.pyEnv["discuss.channel.member"].searchRead([["id", "=", id]]);
+        let target;
```

Comment:
This should also not do anything if we set the existing value for consistency.

Reply:
done :+1: